NOTICE.md

```diff
@@ -7,7 +7,7 @@ Jakarta EE APIs project.
 
 ## Trademarks
 
-Eclipse Project for Stable Jakarta EE APIs is a trademark of the Eclipse
+Eclipse Project for Jakarta Stable APIs is a trademark of the Eclipse
```

Comment:
Just "Jakarta Stable APIs" is a trademark...  Drop the Eclipse Project for...

Reply:
Done